app/src/main/java/org/gdg/frisbee/android/common/GdgActivity.java

```diff
@@ -125,22 +124,23 @@ public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         RecentTasksStyler.styleRecentTasksEntry(this);
 
-        if (!Utils.isEmulator()) {
-            createGoogleApiClient();
-        }
+        mGoogleApiClient = createGoogleApiClient();
```

Comment:
With this, did we lose the ability to work with the app without Play Services installed? Or doesn't it matter?